### PR TITLE
[NP-140, NP-141] feat: improve plan upload and pricing config

### PR DESCRIPTION
**Pull request by ingthanhunt**
## **Ticket**

**JIRA ID:** [NP-140](https://parking-system.atlassian.net/browse/NP-140)
[NP-141](https://parking-system.atlassian.net/browse/NP-140)

## Changes proposed in this pull request
- improve parking request form

please fill all the required fields from below Checklist otherwise reviewer **will not review** this PR
## Checklist
- [ ] Screenshot evidence for both IOS and Android
- [/ ] Total change is less than 500 lines of code

## **Screenshot Evidence**

[NP-140]: https://parking-system.atlassian.net/browse/NP-140?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[NP-141]: https://parking-system.atlassian.net/browse/NP-141?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

https://discord.com/channels/1141381185226948608/1222416635961540698/1228316272949989508

### CODE REVIEW

File: components/parking-lot/ConfigPlan.tsx [hunk 1]
```diff
@@ -1,36 +1,181 @@
...
+    borderWidth: 2,
+    borderColor: Colors.red[400],
+    borderStyle: "dashed",
+    borderRadius: 8,
+    alignItems: "center",
+    justifyContent: "center",
+    paddingVertical: 12,
+    paddingHorizontal: 12,
+  },
+  card: {},
```
Comment (Petoxic) [inline components/parking-lot/ConfigPlan.tsx]: remove it if not use

File: components/parking-lot/ConfigAddress.tsx [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+            style={styles.map}
+            onPress={(event) => {
+              const { latitude, longitude } = event.nativeEvent.coordinate;
+              onChange(event.nativeEvent.coordinate);
+              setRegion({
+                latitude,
+                longitude,
+                latitudeDelta: 0.0522,
+                longitudeDelta: 0.0121,
+              });
```
Comment (TaeCV) [inline components/parking-lot/ConfigAddress.tsx]:
```suggestion
      setRegion((prevRegion: RegionType | undefined) => {
        if (!prevRegion) {
          return {
            latitude: latitude,
            longitude: longitude,
            latitudeDelta: 0.0522,
            longitudeDelta: 0.0121,
          };
        }
        return {
          ...prevRegion,
          latitude: latitude,
          longitude: longitude,
        };
      });
```

File: components/parking-lot/ConfigAddress.tsx [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+            zoomEnabled
+          >
+            <Marker.Animated
+              coordinate={{
+                latitude: region?.latitude || 0,
+                longitude: region?.longitude || 0,
+              }}
+              onDragEnd={(e) => {
+                console.log("dragEnd", e.nativeEvent.coordinate);
+              }}
```
Comment (TaeCV) [inline components/parking-lot/ConfigAddress.tsx]: remove?

File: screens/other/parkinglot/RequestPakingLot.tsx [hunk 4]
```diff
@@ -24,15 +30,25 @@ export type RequestParkingLotProps = CompositeScreenProps<
...
+  const [isSubmitSuccessful, setSubmitSuccessful] = useState<boolean>(false);
+  const { mutateAsync: createRequestAsync } = useCreateParkingLotRequest();
 
-  const onSubmit = async (data: FieldValues) => {
+  const onSubmit = async (data: ParkingLotRequest) => {
     try {
-      // await mutateAsync(data);
-      console.log("data", JSON.stringify(data));
-      navigation.navigate("OtherStack", {screen: "ParkingLotsList"})
+      console.log('data', JSON.stringify(data));
```
Comment (TaeCV) [inline screens/other/parkinglot/RequestPakingLot.tsx]: remove?

**Pull request merged**